### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by ritik-patel05**
https://user-images.githubusercontent.com/67374176/152854501-8bb678e7-8bf3-4acc-bd89-783e99ac9f45.mp4

Inspiration: Bootstrap, Bulma Navbar component

### CODE REVIEW

File: css/form/select.css [hunk 1]
```diff
@@ -24,32 +24,6 @@
...
-
-    position: absolute;
-    top: 50%;
-
-    transform: rotate(-45deg);
-    transform-origin: center;
-
-    border-color: var(--primary);
-    right: 1.125em;
-    z-index: 4;
```
Comment (ZuberDunge) [inline css/form/select.css:50]: remove extra lines

**Review by ZuberDunge — COMMENTED** [review]
code is well organised op

File: css/elements/button.css [hunk 2]
```diff
@@ -198,3 +198,13 @@
...
+
+/* ---- Group Of Buttons ---- */
+
+.buttons .btn {
+    margin-bottom: var(--size-5);
+}
+
+.buttons .btn:not(:last-child) {
+    margin-right: var(--size-5);
+}
```
Comment (hinanshisuthar) [inline css/elements/button.css]: Add a new line at the EOF

File: css/components/navbar.css [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+    .navbar .navbar-menu {
+        flex-shrink: 1;
+    }
+
+    .navbar .navbar-menu, 
+    .navbar .navbar-start,
+    .navbar .navbar-end {
+        align-items: center;
+    }
+}
```
Comment (hinanshisuthar) [inline css/components/navbar.css]: Hero, too.

**Review by hinanshisuthar — COMMENTED** [review]
Everything else looks perfectly fine.

File: css/form/select.css (hunk 1 above)
Comment (ritik-patel05) [reply css/form/select.css:50]: Its removed code

File: css/elements/button.css (hunk 2 above)
Comment (ritik-patel05) [reply css/elements/button.css]: Its already there, else it would show huge circle dot with a line in it.

**Pull request merged**